utils/pvacapi/controllers/database.py

```diff
@@ -323,7 +330,7 @@ def fileschema(parentID, fileID):
     )
 
     # check if the table exists:
-    db = current_app.config['storage']['db']
+    db = psql.open("localhost/pvacseq")
```

Comment:
Is it correct to use `localhost` here? We've been making some changes so that people can run pvacapi with any arbitrary ip address (`--ip-address` argument).

Reply:
It should be correct to use `localhost` here.  This line of code creates a connection to the locally run postgres database server, which is run at `localhost/pvacseq`.  As of the current version of staging, this is how the database is setup to run in `utils.py`.

Reply:
Ah, you're correct. Thanks for the clarification. I still would like to see the refactor to move some of the code into their own subroutines, if possible but otherwise +1